api/src/main/scala/Geoprocessing.scala

```diff
@@ -67,8 +67,13 @@ trait Geoprocessing extends Utils {
             case "RasterLinesJoin" =>
               input.streamLines match {
                 case Some(mls) => {
-                  val lines = (parseMultiLineString(mls) & shape).asMultiLine.get
```

Comment:
It's interesting that Scala makes `.get` available at all since the failure case is inherently unhandled.

Reply:
Documentation does state that it throws an exception: https://www.scala-lang.org/api/current/scala/Option.html#get:A

Maybe had there not been any exception handling in the chain there may have been some compiler warnings, but since AKKA HTTP does handle all exceptions automatically (returning a 500 and so forth) we never got it.